packages/one-app-bundler/CHANGELOG.md

```diff
@@ -138,3 +148,142 @@
 
 
 
+
+
+# 6.12.0 (2021-09-13)
```

Comment:
I was unable to figure out the reason for duplicate changelog updates. One possible reasons is incorrectly annotated tags due to the transition to independent versioning